src/js/popup.js

```diff
@@ -195,7 +195,11 @@ function limitAccess (callback = false) {
                     $('#customTemplateIssueTypeField').prop('disabled', true);
                     $('#customTemplateProjectsField').prop('disabled', true);
                     $('#addDefaultDropdownButton').addClass('disabled');
-                    $('#customDomains').addClass('disabled');
+                    $('#customSettings').addClass('disabled');
+                    $('#customDomainInputButton').addClass('disabled');
+                    $('#clearCustomDomains').addClass('disabled');
```

Comment:
I think we're missing the individual delete buttons too. 
For something like this, it would make sense to have a class that's attached to all the ui elements we want to be able to disable. Something along the lines of: `custom-domains`/`custom-inputs`